app/models/experimentation/current_patient_experiment.rb

```diff
@@ -12,9 +12,12 @@ def eligible_patients(date)
         .distinct
     end
 
-    def memberships_to_notify(date)
-      # Patients where `date` equals one of their reminder template's remind_on.
-      # To be implemented in a follow up PR.
+    # Patients whose expected return date falls on
+    # one of the reminder template's remind_on days since `date`.
+    def memberships_to_notify(reminder_template, date)
```

Comment:
I think I asked this in a different PR earlier: can we call this `patients_to_notify`? We don't actually notify the "Membership", which is a relationship between the patient and experiment.

Reply:
Could also consider `members_to_notify`.

Reply:
I'm wary of calling it `patients_to_notify`, since this returns a `TreatmentGroupMembership` relation. The same for `members_to_notify`, we don't have the idea of a "member" anywhere. A "members" relation would ideally just be a `Patient` relation.

With the changes in 0c6a229d8, calling it `memberships` seems more appropriate, however it doesn't reflect that the result has a join with `reminders` 😅 . wdyt?